java/sync-map-and-scene-viewpoints/README.md

```diff
@@ -1,20 +1,35 @@
-# Sync Map and Scene Viewpoints
-This sample demonstrates how to keep the viewpoints of multiple map or scene views in sync, so that navigating one view immediately updates the others.
+# GeoView viewpoint synchronization
```

Comment:
I think we agreed not to change sample titles for now if they deviate from the original name: since we'd also then have to update the code class names etc. The names will be revisited at a later date I believe.

Reply:
sorry I was a bit distracted friday :(